src/tags/index.js

```diff
@@ -129,23 +129,68 @@ module.exports = function(fractal){
                 }
             }
         },
+
         trans(Twig) {
             return {
                 type: 'trans',
-                regex: /^trans/,
+                regex: /^trans(.+)?/,
                 next: ['endtrans'],
```

Comment:
The trans/plural/endtrans tag combo is very similar to an if/else/elseif tag combo – shouldn't we specify `next` and the other properties just like in there? https://github.com/twigjs/twig.js/blob/master/src/twig.logic.js#L122-L128

Reply:
We can add it, but it works in both ways.